src/arcade/patch/agent/process/PatchProcessMetabolismMedium.java

```diff
@@ -47,6 +47,9 @@ public class PatchProcessMetabolismMedium extends PatchProcessMetabolism {
     /** Rate of ATP production [fmol ATP/um<sup>3</sup>/min/M glucose]. */
     private final double atpProductionRate;
 
+    /** Initial cell internal glucose concentration [fmol]. */
+    private final double initGluc;
```

Comment:
Since `initGluc` is only used in the constructor, I think you can just inline the variable

Reply:
fixed !